package.json

```diff
@@ -50,6 +50,6 @@
     "webpack": "3 || 4 || 5"
   },
   "engines": {
-    "node": ">=v12.22.12"
+    "node": ">=14.18.0"
```

Comment:
(we might upgrade to higher version in next major, will have to check other deps)